static/app/utils/fields/index.ts

```diff
@@ -403,6 +403,46 @@ export enum IsFieldValues {
   UNLINKED = 'unlinked',
 }
 
+enum IsFieldDescriptions {
+  RESOLVED = 'Issues marked as fixed',
```

Comment:
strings should be translated with our t function
```suggestion
  RESOLVED = t('Issues marked as fixed'),
```